changelog/fix_some_false_positives_by_replacing.md

```diff
@@ -0,0 +1 @@
+* [#866](https://github.com/rubocop/rubocop-rails/pull/866): Fix some false-positives by replacing `const _` with `const {nil? cbase}`. ([@r7kamura][])
```

Comment:
Can you tweak it? e.g.:

```suggestion
* [#866](https://github.com/rubocop/rubocop-rails/pull/866): Make `Rails/DurationArithmetic`, `Rails/IndexBy`, `Rails/IndexWIth`, and `Rails/RequireDependency` cops aware of const starts with `::`. ([@r7kamura][])
```

Reply:
Thank you, the recommended example is more user-friendly and easier to understand 👍